spec/requests/create_bookmark_from_email_spec.rb

```diff
@@ -13,12 +13,23 @@
     expect(@user.topics.first.bookmarks).to match_array [Bookmark.find_by(url: "www.google.com")]
     expect(response.status).to eq(200) 
   end
-  xit "creates a bookmark if topic already exists" do
+  it "creates a bookmark if topic already exists" do
+    topic = create(:topic, title: "animals", user: @user)
+    post '/bookmarks',{sender: "test@example.com", subject: "animals", "body-plain": "www.google.com"}
+    expect(@user.topics.first.bookmarks.first).to belong_to(:topic)
```

Comment:
I think the `belong_to(:topic)` test here isn't testing what's intended. Check for yourself by commenting out line 17. The test will still pass without line 17 I suspect.